CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+
+## v2.0.0 (WIP)
+
+- Refines api call order to getToken -> getProvider -> GitHub Operations instead of the previous getProvider -> getToken
```

Comment:
Advice to use the convention of only using one of the following verbs as the first word of the release note:

> - Added – For any new features that have been added since the last version was released
> - Changed – To note any changes to the software’s existing functionality
> - Deprecated– To note any features that were once stable but are no longer and have thus been removed
> - Fixed– Any bugs or errors that have been fixed should be so noted
> - Removed– This notes any features that have been deleted and removed from the software
> - Security– This acts as an invitation to users who want to upgrade and avoid any software vulnerabilities
>
> ~~<https://changelog.md>~~ Seems like that website is down at the moment, but Internet Archive got us covered: http://web.archive.org/web/20210531040247/https://changelog.md/